Examples/Modules/laser_injection_from_file/inputs.3d_test_txye

```diff
@@ -0,0 +1,56 @@
+#################################
+####### GENERAL PARAMETERS ######
+#################################
+max_step = 175
+amr.n_cell =  384 112 112
+amr.max_grid_size = 64
+amr.blocking_factor = 16
+amr.max_level = 0
+geometry.coord_sys   = 0                  # 0: Cartesian
+geometry.is_periodic = 1 1 1     # Is periodic?
+geometry.prob_lo     = -10.e-6  -5.0e-6  -5.0e-6     # physical domain
+geometry.prob_hi     = 10.e-6   5.e-6    5.e-6
+warpx.verbose = 1
+warpx.serialize_ics = 1
+
+#################################
+############ NUMERICS ###########
+#################################
+interpolation.nox = 3
+interpolation.noy = 3
+interpolation.noz = 3
+warpx.cfl = 0.98
+warpx.do_dynamic_scheduling = 0
+warpx.load_balance_int = -1
+warpx.use_filter = 0
+algo.maxwell_fdtd_solver = ckc
+
+#################################
+############ PLASMA #############
+#################################
+particles.nspecies = 0
+
+#################################
+############# LASER #############
+#################################
+lasers.nlasers      = 1
+lasers.names        = txye_laser
+txye_laser.position     = 0. 0. 0. # This point is on the laser plane
+txye_laser.direction    = 1. 0. 0.     # The plane normal direction
+txye_laser.polarization = 0. 1  0.     # The main polarization vector
+txye_laser.e_max        = 1.e12        # Maximum amplitude of the laser field (in V/m)
+txye_laser.wavelength = 1.0e-6         # The wavelength of the laser (in meters)
+txye_laser.profile      = from_txye_file
+txye_laser.txye_file_name = "gauss_3d_unf.txye"
+txye_laser.time_chunk_size = 50
+
+# Diagnostics
+diagnostics.diags_names = diag_full diag_reduced
+diag_full.period = 175
+diag_full.diag_type = Full
+diag_full.fields_to_plot = Ey
```

Comment:
Do we still need the high-res output?

Reply:
The analysis is done on the high resolution output, but the script deletes everything except the last low-resolution output:

```
os.system("./" + executable + " inputs.3d_test_txye diag_full.file_prefix=diags/plotfiles/plt diag_reduced.file_prefix=diags/plotfiles/rplt")

do_analysis_3d("diags/plotfiles/plt00175/", "comp_unf.pdf", 175)

os.system("sed 's/gauss_3d_unf.txye/gauss_3d.txye/g' inputs.3d_test_txye > inputs.3d_test_txye_non_unf")

os.system("./" + executable + " inputs.3d_test_txye_non_unf diag_full.file_prefix=diags/plotfiles/plt diag_reduced.file_prefix=diags/plotfiles/rplt")
do_analysis_3d("diags/plotfiles/plt00175/", "comp_non_unf.pdf", 175)

os.system("rm -r diags/plotfiles/rplt00000*") #delete low resolution plots at 0

os.system("rm -r diags/plotfiles/plt*") #delete all high resolution plots
```